src/RA_Core.cpp

```diff
@@ -905,6 +916,7 @@ static void RA_CheckForUpdate()
     std::string Response;
     if (RAWeb::DoBlockingRequest(RequestLatestClientPage, args, Response))
     {
+        // TODO: this doesn't work - response is JSON
```

Comment:
this code just isn't working right now. "Check for Updates" menu option. it's expecting a single string instead of a JSON package